### PR TITLE
Added 1.0 Blocks

### PR DESCRIPTION
**Pull request by FunnyBuddys**
Hello Guys, i added some Blocks from 1.0.0 

Items will i change in another Pull request.

### CODE REVIEW

File: src/pocketmine/block/ChorusPlant.php [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+	public function getName() : string{
+		return "Chorus Plant";
+	}
+
+	public function getDrops(Item $item) : array {
+		$drops = [];
+		if($this->meta >= 0x07){
+			$fortunel = $item->getEnchantmentLevel(Enchantment::TYPE_MINING_FORTUNE);
+			$fortunel = $fortunel > 3 ? 3 : $fortunel;
+			$drops[] = [Item::CHORUS_FRUIT, 0, 1];
```
Comment (FunnyBuddys) [inline src/pocketmine/block/ChorusPlant.php]: Chorus Fruit will i add Later!

**Comment by JunkDaCoder** [comment]
Fortune doesnt effect chorus fruit though...

**Comment by SuperMaXAleX** [comment]
I'll merge it when there are going to have every new blocks and items ! :)

**Comment by FunnyBuddys** [comment]
Ok! Thanks!

**Pull request closed without merging**